Packs/CommonTypes/Layouts/layout-indicatorsDetails-Account.json

```diff
@@ -514,31 +514,21 @@
                         "y": 3
                     },
                     {
-                        "displayType": "ROW",
                         "h": 2,
-                        "hideItemTitleOnlyOne": true,
-                        "hideName": false,
-                        "i": "default-main-69a31ec0-d0d9-11ea-937a-6f8841d9fed1",
-                        "items": [
-                            {
-                                "endCol": 2,
-                                "fieldId": "feedrelatedindicators",
-                                "height": 22,
-                                "id": "74cd8c90-d0d9-11ea-937a-6f8841d9fed1",
-                                "index": 0,
-                                "sectionItemType": "field",
-                                "startCol": 0
-                            }
-                        ],
+                        "i": "default-main-302ebde0-dbb3-11ea-894b-59859b6b5482",
+                        "items": [],
                         "maxW": 3,
                         "minH": 1,
                         "minW": 1,
                         "moved": false,
                         "name": "Feed Related Indicators",
+                        "query": "9d218728-3464-4404-8e7d-07fb28283734",
```

Comment:
isnt this supposed to be the script id ?

Reply:
This is how it download it from server

Reply:
I'll change it manually to the script id.